src/lib/data/index.ts

```diff
@@ -122,8 +122,6 @@ export function useAppConfig(): QueryResult<AppConfigQuery, AppConfigQueryVariab
 
 type TrackingEvent = 'Click' | 'View' | 'Scroll' | 'Submit';
 type TrackingObject =
-    | 'Content'
```

Comment:
Might need to check heap to see if we are already calling these `Content` there, and if so we'll either need to keep as Content or do something to merge the events together.

Reply:
Good call!